### PR TITLE
Feature/group user list progress

### LINKED ISSUE #401: On group/user page: list progress
## Motivations
List group progress on group the current-user can watch.

## Subtasks
- On the group page in the "overview tab", and other user's page, display a table presenting a log of the recent progress of the group/user IF the current user can watch the group/user. 
- use [itemActivityLogForAllItems](https://france-ioi.github.io/algorea-devdoc/api/#operation/itemActivityLogForAllItems) service, passing as `watched_group_id` the group/user_id.
- get inspired or reuse the item-log-view component

### PR DESCRIPTION
**Pull request by Iloveall**
Fixes #401

### CODE REVIEW

**Review by smadbe — CHANGES_REQUESTED** [review]
First quick remark: If I go on the progress on the usual user and click on "Empty chapter", I arrive on an error page (while this page is visible by the user), so I assume there is an error in the links somehow (if you think the issue is not related with your implementation, let me know).
Screenshot 2021-07-05 at 12 52 19

**Review by smadbe — CHANGES_REQUESTED** [review]
Only small details to fix.

File: src/app/modules/group/components/group-log-view/group-log-view.component.ts [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+        header: $localize`User`,
+        disabled: !this.showUserColumn,
+      },
+      {
+        field: 'at',
+        header: $localize`Time`,
+      }
+    ];
+
+    return columns.filter(item => !item.disabled).map(item => ({
```
[the remark this answers is not in the document]
Comment (Iloveall) [reply src/app/modules/group/components/group-log-view/group-log-view.component.ts]: Fixed

**Review by smadbe — APPROVED** [review]
👍 Maybe wait for the other PR to be ready before merging it.

File: src/app/modules/group/components/group-log-view/group-log-view.component.html [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+              <ng-container *ngSwitchCase="'activityType'">
+                {{ rowData.activityType | logActionDisplay : rowData.score }}
+              </ng-container>
+              <ng-container *ngSwitchCase="'item.string.title'">
+                <a class="alg-link" [ngClass]="{'disabled': !rowData.item}" [routerLink]="rowData.item | itemLink">
+                  {{ rowData.item.string.title }}
+                </a>
+              </ng-container>
+              <ng-container *ngSwitchCase="'item.user'">
+                {{ rowData.user | userCaption }}
```
[the remark this answers is not in the document]
Comment (Iloveall) [reply src/app/modules/group/components/group-log-view/group-log-view.component.html]: Fixed

**Pull request merged**